x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -93,48 +89,82 @@ export const AnalyticsPanel: FC<Props> = ({
 
   const panelClass = isInitialized === false ? 'mlOverviewPanel__isLoading' : 'mlOverviewPanel';
 
+  const transformsLink = `${basePath.get()}/app/management/data/transform`;
+
+  const noDFAJobs = errorMessage === undefined && isInitialized === true && analytics.length === 0;
+
+  if (noDFAJobs) {
+    return (
+      <EuiEmptyPrompt
+        layout="horizontal"
+        hasBorder={true}
+        hasShadow={false}
+        icon={<EuiImage size="fullWidth" src={adImage} alt="anomaly_detection" />}
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.overview.analyticsList.createFirstJobMessage"
+              defaultMessage="Create your first data frame analytics job"
+            />
+          </h2>
+        }
+        body={
+          <>
+            <p>
+              <FormattedMessage
+                id="xpack.ml.overview.analyticsList.emptyPromptText"
+                defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis and put the annotated data in a new index. The classification and regression trained models can also be used for inference in pipelines and aggregations."
+              />
+            </p>
+            <EuiCallOut
+              size="s"
+              title={
+                <FormattedMessage
+                  id="xpack.ml.overview.analyticsList.emptyPromptHelperText"
+                  defaultMessage="Data frame analytics requires specifically structured source data. Use {transforms} to create data frames before you create the jobs."
+                  values={{
+                    transforms: (
+                      <EuiLink href={transformsLink} target="blank" color={'accent'}>
+                        <FormattedMessage
+                          id="xpack.ml.overview.gettingStartedSectionTransforms"
+                          defaultMessage="Elasticsearch's transforms"
+                        />
+                      </EuiLink>
+                    ),
+                  }}
+                />
+              }
+              iconType="iInCircle"
+            />
+          </>
+        }
+        actions={
+          <EuiButton
+            onClick={() => setIsSourceIndexModalVisible(true)}
+            color="primary"
+            fill
+            isDisabled={jobCreationDisabled}
+            data-test-subj="mlOverviewCreateDFAJobButton"
+          >
+            <FormattedMessage
+              id="xpack.ml.overview.analyticsList.createJobButtonText"
+              defaultMessage="Create job"
```

Comment:
Clicking the `Create job` button doesn't seem to be having any effect. 
![image](https://user-images.githubusercontent.com/6446462/146262736-efaac5ff-c075-4ec6-b29b-730230dabdc7.png)

Reply:
Thanks for spotting this @alvarezmelissa87 , fixed in e4e6769b8970fb764c744bc0d2dd6b1cac56c826v.
By the way, I think we should align DFA jobs with the AD and navigate to a dedicated route for the job wizard.